src/Tribe/Attendees.php

```diff
@@ -558,6 +558,8 @@ private function generate_filtered_list( $event_id ) {
 
 				// Handle custom columns that might have names containing HTML tags
 				$row[ $column_id ] = wp_strip_all_tags( $row[ $column_id ] );
+				//Decode HTML Entities
```

Comment:
```suggestion
				// Decode HTML Entities.
```
There must be a space between the `//` and the comment. All comments need to have a stop at the end.